include/6DA460.h

```diff
@@ -2,14 +2,15 @@
 #define C_6DA460_H
 
 #include "ultra64.h"
+#include "m_mail.h"
 
 struct Game_Play;
 
 // void func_800B67C0_jp();
 // void func_800B6838_jp();
-// void func_800B68E8_jp();
+s32 mPO_get_keep_mail_sum(void);
 // void func_800B690C_jp();
-// void func_800B6A3C_jp();
+s32 mPO_receipt_proc(Mail_c*, s32);
```

Comment:
Instead of including `m_mail.h` could you forward reference `Mail_c` and use that?

(github doesn't allow me to make a suggestion :c )